db/v-2/insert-test-data/user-insert.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="user-insert" author="petertrr" context="dev">
+        <insert dbms="mysql" tableName="user">
+            <column name="id" value="1" />
+            <column name="name" value="admin" />
+            <column name="password" value="{bcrypt}$2a$12$7jB6.fJ/uHZiti94w0.QOu0M/lPzKU1lvX9TLL3uZHFjLGIfz1gHq" />
```

Comment:
{bcrypt}?

Reply:
That's the format for password storage in Spring Security 5. Encoder can decode passwords in different formats, but the new ones are Encoder with bcrypt.